cli/cmd/gnutls.go

```diff
@@ -35,17 +37,27 @@ ecapture gnutls
 ecapture gnutls --hex --pid=3423
 ecapture gnutls -l save.log --pid=3423
 ecapture gnutls --gnutls=/lib/x86_64-linux-gnu/libgnutls.so
+ecapture gnutls -m keylog -k ecapture_gnutls_key.og --ssl_version=3.7.9
```

Comment:
Please change `ecapture_gnutls_key.og` to `ecapture_gnutls_key.log`. The previous writing was a spelling mistake.

Reply:
the part is finished in https://github.com/gojue/ecapture/pull/654/commits/f514903293dfc1d1afe245a766b02bf323b8b8d5